src/material-experimental/mdc-slider/slider-thumb-decorator.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  Output,
+  ViewChild,
+  ViewEncapsulation,
+} from '@angular/core';
+import {Thumb} from '@material/slider';
+
+/**
+ * Handles displaying the slider knobs and their value indicators.
+ */
+@Component({
+  selector: 'mat-slider-start-thumb-decorator, mat-slider-end-thumb-decorator',
+  templateUrl: 'slider-thumb-decorator.html',
+  host: {
+    'class': 'mdc-slider__thumb',
+    '(mouseenter)': '_mouseenter.emit()',
+    '(mouseleave)': '_mouseleave.emit()',
```

Comment:
Would these need to account for touch devices?

Reply:
Probably. Would it be acceptable for me to add a TODO here and implement the support for this in a later PR?